test/cli.js

```diff
@@ -7,3 +7,17 @@ test('main', async t => {
 		/Missing Awesome badge/
 	);
 });
+
+test('main - non-existent file', async t => {
+	await t.throwsAsync(
+		execa.stderr('./cli.js', ['test/fixtures/non-exsitent.md']),
```

Comment:
can you please rename this file to `non-existent.md`? just a small spelling mistake

Reply:
@transitive-bullshit Done.